code/modules/mob/living/simple_animal/bot/cleanbot.dm

```diff
@@ -98,6 +98,14 @@
 	if(emagged == 2) //Emag functions
 		if(istype(loc,/turf/simulated))
 
+			for(var/mob/living/carbon/victim in src.loc)
+				if(victim.stat != DEAD)//cleanbots always finish the job
+					victim.visible_message("<span class='danger'>[src] sprays hydrofluoric acid at [victim]!</span>", "<span class='danger'>[src] sprays you with hydrofluoric acid!</span>")
+					victim.adjustFireLoss(25)

```

Comment:
acid as a reagent is basically the equivalent of urine in terms of lethality, but i want the emag function to actually do something.